lib/active_model/serializer/utils.rb

```diff
@@ -23,7 +42,9 @@ def include_args_to_hash(included)
     when Symbol
       { included => {} }
     when Hash
-      included.each_with_object({}) { |(key, value), hash| hash[key] = include_args_to_hash(value) }
```

Comment:
I thought we were doing the semantic usage?  But I guess Rails does multiline, so we should probably switch to Rails-style here..  would be a simple PR, and a change in rubocop config to disable the semantic block params